src/LodeRunner.API/src/Handlers/RequestLogger/RequestLogger.cs

```diff
@@ -209,10 +213,33 @@ private static string GetClientIp(HttpContext context, out string xff)
             // Note: clientIp will be null when running Test Project
             // remove IP6 local address
             return clientIp?.Replace("::ffff:", string.Empty);
+        }
+
+        /// <summary>
+        /// Extracts and returns request body
+        /// </summary>
+        /// <param name="request">HttpRequest</param>
+        /// <returns>Request body as object.</returns>
+        private static object LogRequestPayload(HttpRequest request)

```

Comment:
the method does not actually log, I think should be named more like GetPayloadRequestObject or something around that.

Reply:
Updated the function name